docs/source/usage/openshift.rst

```diff
@@ -0,0 +1,76 @@
+OpenShift
+=========
+
+Owners and lessees can easily install OpenShift on hardware managed through ESI through the use of the `OpenShift assisted installer`_. This guide assumes that the relevant roles have been granted access to the required Ironic actions.
+
+Prerequisites
+-------------
+
+* Access to OpenShift assisted installer
+   * You may use https://console.redhat.com or a local installation of the OpenShift assisted installer
+* An HTTP server for the OpenShift ISO generated by the assisted installer
+* Usage of ESI-managed nodes in the 'available' state
+
+The Ironic provisioning network must also be able to reach the HTTP server and the assisted installer. If you are using https://console.redhat.com, the following commmands will provide the appropriate access:
+
+  .. prompt:: bash $
+
+    openstack subnet set --dns-nameserver 8.8.8.8 <provisioning subnet>
+    sudo iptables -t nat -A POSTROUTING -s <provisioning subnet cidr> -j MASQUERADE
+
+Prepare Cluster on Assisted Installer
+-------------------------------------
+
+* Log onto the assisted installer and select 'OpenShift' from the menu
+* Click on 'Create Cluster', select the 'Datacenter' tab, and create a cluster from the 'Assisted Installer' subsection
+* Input 'Cluster details' and click 'Next'
+   * If you're only using a single node, click the 'Install single node OpenShift (SNO)' checkbox
+* In the 'Host discovery' step, choose 'Generate Discovery ISO'
+   * Select 'Minimal image file: Provision with virtual media'
+   * Input your SSH public key
+   * Download the ISO onto your HTTP server
+
+Configure Ironic Nodes
+----------------------
+
+* Run these commands to configure your nodes to boot from the generated OpenShift ISO:
+
+  .. prompt:: bash $
+
+    openstack baremetal node set --instance-info deploy_interface=ramdisk <node>
+    openstack baremetal node set --instance-info boot_iso=<openshift iso url> <node>
+
+* Attach the node to Ironic's provisioning network:
+
+  .. prompt:: bash $
+
+    openstack esi node network attach --network <provisioning network> <node>
+
+
+Deploy
+------
+
+* Once your nodes are ready, run the deploy command:
+
+  .. prompt:: bash $
+
+    openstack baremetal node deploy <node>
+
+* Wait for the nodes to appear on the assisted installer as Ready. Click Next.
+* In the 'Networking' step, select the appropriate subnet. Click Next.
+   * You can switch your node's networks if needed by running these commands and waiting for the subnet list to refresh:
+
+  .. prompt:: bash $
+
+    openstack esi node network list    # find names of ports attached to nodes
+    openstack esi node network detach <node> <port>
+    openstack esi node network attach --network <other network> <node>
+* Installation will continue and eventually complete. Once it does, the assisted installer will have credentials for logging into your OpenShift console.
```

Comment:
I think you're missing the steps necessary to actually start the install (in the assisted installer ui)? Your last instruction is "In the 'Networking' step, select the appropriate subnet. Click Next.", and I think there's something after that before the install actually starts.

Reply:
Actually, there isn't! Clicking "Next" after selecting the subnet will start the installation.